ocaml-lsp-server/src/compl.ml

```diff
@@ -31,16 +31,20 @@ let completion_kind kind : CompletionItemKind.t option =
 
 let make_string chars =
   let chars = Array.of_list chars in
-  String.init (Array.length chars) ~f:(Array.get chars)
+  let arr_len = Array.length chars in
+  (String.init (Array.length chars) ~f:(Array.get chars), arr_len)
```

Comment:
Why do we return the length here? The caller can always obtain it with `String.length`

Reply:
As `make_string` anyway computes the length of the string, I thought why to do that twice.